pkg/tfgen/docs.go

```diff
@@ -1767,6 +1764,34 @@ func (g *Generator) convertHCL(e *Example, hcl, path string, languages []string)
 	isCompleteFailure := len(failedLangs) == len(languages)
 
 	if isCompleteFailure {
+		languagesByErrMsg := map[string]map[string]struct{}{}
+		for lang, convertErr := range failedLangs {
+			m := convertErr.Error()
+			if _, ok := languagesByErrMsg[m]; !ok {
+				languagesByErrMsg[m] = map[string]struct{}{}
+			}
+			languagesByErrMsg[m][lang] = struct{}{}
+		}
+
+		var err error
+
+		seen := map[string]struct{}{}
```

Comment:
Why does this map need to be mapping to a struct?

Reply:
This is a classic go representation of Sets. Unfortunately there is no Set in stdlib like in Python.